upload/lambdas/checksum_daemon/checksum_daemon.py

```diff
@@ -39,6 +39,8 @@ def __init__(self, context):
         logger.debug("Ahm ahliiivvve!")
         self.config = UploadConfig()
         self._read_environment()
+        self.upload_service_version = UploadVersion().upload_service_version
+        logger.debug("UPLOAD_SERVICE_VERSION: {}".format(self.upload_service_version))
```

Comment:
It's good that we log the version when the checksum daemon lambda starts.
It would also be good to log it when the API lambda starts.
I can't see an easy way to do that, however.  If you can, let's do it. Otherwise nevermind :)

Reply:
I dont know if there is a straightforward way to do that. I could log it on every request, or as part of the health check